lib/Runtime/Language/JavascriptOperators.cpp

```diff
@@ -9489,7 +9489,7 @@ namespace Js
 
         if (e != nullptr)
         {
-            reject->GetEntryPoint()(reject, CallInfo(CallFlags_Value, 2), library->GetUndefined(), e->GetThrownObject(scriptContext));
+            JavascriptPromise::TryRejectWithExceptionObject(e, reject, scriptContext);
```

Comment:
Thanks for the heads-up. I have fixed all the other places TryRejectWithExceptionObject is called. Can you take a look?